site/pages/create-consequence-stops/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -291,7 +291,7 @@ const CreateConsequenceStops = (props: CreateConsequenceStopsProps): ReactElemen
                                     props.disruptionSummary ? stateUpdater(props.disruptionSummary, "description") : "";
```

Comment:
Can we change the name of this prop to disruptionDescription to reflect it is copy the description not a summary

Reply:
done